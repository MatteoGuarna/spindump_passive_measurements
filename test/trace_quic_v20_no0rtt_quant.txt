A QUIC run without 0-RTT usage
