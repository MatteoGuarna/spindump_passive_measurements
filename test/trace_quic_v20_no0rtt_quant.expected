[
{ "Event": "new", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "04c7937f-2ff1adbf3baa49026434ad9a3aae (55154:4433)", "Ts": 1558426581987957, "State": "Starting", "Packets1": 0, "Packets2": 0, "Bytes1": 0, "Bytes2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "04c7937f-2ff1adbf3baa49026434ad9a3aae (55154:4433)", "Ts": 1558426582012041, "State": "Starting", "Right_rtt": 24084, "Packets1": 1, "Packets2": 0, "Bytes1": 1228, "Bytes2": 0, "Bandwidth1": 1228, "Bandwidth2": 0 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "04c7937f-2ff1adbf3baa49026434ad9a3aae (55154:4433)", "Ts": 1558426582014712, "State": "Starting", "Left_rtt": 2671, "Packets1": 1, "Packets2": 1, "Bytes1": 1228, "Bytes2": 60, "Bandwidth1": 1228, "Bandwidth2": 60 },
{ "Event": "change", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "04c7937f-04c331dcd37fa266 (55154:4433)", "Ts": 1558426582040560, "State": "Starting", "Packets1": 2, "Packets2": 1, "Bytes1": 2456, "Bytes2": 60, "Bandwidth1": 2456, "Bandwidth2": 60 },
{ "Event": "measurement", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "04c7937f-04c331dcd37fa266 (55154:4433)", "Ts": 1558426582040560, "State": "Up", "Right_rtt": 25848, "Packets1": 2, "Packets2": 1, "Bytes1": 2456, "Bytes2": 60, "Bandwidth1": 2456, "Bandwidth2": 60 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "d24205a0-04c331dcd37fa266 (55154:4433)", "Ts": 1558426582040571, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 2, "Packets2": 4, "Bytes1": 2456, "Bytes2": 3538, "Bandwidth1": 2456, "Bandwidth2": 3538 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "063bc98d-04c331dcd37fa266 (55154:4433)", "Ts": 1558426582068209, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 3, "Packets2": 6, "Bytes1": 2859, "Bytes2": 4130, "Bandwidth1": 2859, "Bandwidth2": 4130 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "063bc98d-04c331dcd37fa266 (55154:4433)", "Ts": 1558426582069179, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 3, "Packets2": 7, "Bytes1": 2859, "Bytes2": 4190, "Bandwidth1": 2859, "Bandwidth2": 4190 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "063bc98d-04c331dcd37fa266 (55154:4433)", "Ts": 1558426582069183, "State": "Up", "Value": 0, "Who": "responder", "Packets1": 3, "Packets2": 8, "Bytes1": 2859, "Bytes2": 5470, "Bandwidth1": 2859, "Bandwidth2": 5470 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "063bc98d-04c331dcd37fa266 (55154:4433)", "Ts": 1558426582071266, "State": "Up", "Value": 1, "Who": "initiator", "Packets1": 3, "Packets2": 9, "Bytes1": 2859, "Bytes2": 6374, "Bandwidth1": 2859, "Bandwidth2": 6374 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "063bc98d-04c331dcd37fa266 (55154:4433)", "Ts": 1558426582071778, "State": "Up", "Value": 1, "Who": "initiator", "Packets1": 4, "Packets2": 9, "Bytes1": 2969, "Bytes2": 6374, "Bandwidth1": 2969, "Bandwidth2": 6374 },
{ "Event": "spinvalue", "Type": "QUIC", "Addrs": ["192.168.203.236","52.58.13.57"], "Session": "063bc98d-04c331dcd37fa266 (55154:4433)", "Ts": 1558426582092879, "State": "Up", "Value": 1, "Who": "responder", "Packets1": 5, "Packets2": 9, "Bytes1": 3027, "Bytes2": 6374, "Bandwidth1": 3027, "Bandwidth2": 6374 }
]
